### ISSUE TITLE
Liquid syntax error: Unknown tag 'cite'

### ISSUE THREAD

**Issue by balazsdukai** [inukshuk/jekyll-scholar#162]
I've set up the Jekyll-Scholar plugin by following the instruction in the README and as a result in my `Gemfile` I have:

```
# If you have any plugins, put them here!
gem "jekyll-scholar"
```

furthermore in the `_config.yml` I put:

```
# Load plugins
gems: ['jekyll/scholar']

scholar:
  style: elsevier-with-titles-alphabetical
  locale: en

  sort_by: none
  order: ascending

  group_by: none
  group_order: ascending

  source: ./_bibliography
  bibliography: report.bib
  bibliography_template: "{{reference}}"

  replace_strings: true
  join_strings:    true

  use_raw_bibtex_entry: false
  bibtex_filters:
  - superscript
  - latex

  details_dir:    bibliography
  details_layout: bibtex.html
  details_link:   Details

  query: "@*"
```

After these steps I ran `bundle install` to install the plugin.

Then I have citations in my document in the format of e.g. `{% cite heijer2012campus %}`. However, when I run `bundle exec jekyll serve` I get the error message `Liquid Exception: Liquid syntax error (line 22): Unknown tag 'cite' in index.md`, where line 22 is simply the first occurrence of a citation with the `cite` tag.

As I am using Ruby and Jekyll the first time, I am not sure if this is an issue or I've just missed something in the configuration files.

**Comment by inukshuk** [comment]
The configuration seems alright to me. You could check if jekyll-scholar was installed correctly, e.g. run `bundle show jekyll-scholar`. Make a note of the directory that is shown. Then you could try and make sure that it is actually being loaded, for example by putting some code into jekyll-scholar's entry point at `lib/jekyll/scholar.rb` relative to where the Gem is installed (the directory from the previous step). You could put something like `puts 'here'` on the first line of that file. Then, when you run jekyll and jekyll-scholar is actually loaded it should print 'here' somewhere to your console (alternatively you could put something that is not Ruby there and see if jekyll fails).

**Comment by R-Gaurav** [comment]
Got this issue fixed from the solution presented in above #194 by making sure that my `Gemfile` contains `gem "jekyll-scholar", group: :jekyll_plugins` line. Actually `group: :jekyll_plugins` was missing earlier.
